docs/ecosystem.md

```diff
@@ -38,20 +39,20 @@ The following document provides an ecosystem overview of how the various tools c
 
 ## Supported quantum cloud services
 
-|Company|Description|
+|Company|Type of Quantum Computer Supported|
 |--- |--- |
-|[Google QCS](https://quantumai.google/cirq/tutorials/google/start)|A quantum computing service based on superconducting qubits|
-|[Alpine Quantum Technologies](https://quantumai.google/cirq/tutorials/aqt/start)|A quantum computing service based on ion traps|
-|[Pasqal](https://quantumai.google/cirq/tutorials/pasqal/start)|A quantum cloud service based on neutral atoms|
-
+|[Google QCS](https://quantumai.google/cirq/tutorials/google/start)|Superconducting qubits|
+|[Alpine Quantum Technologies](https://quantumai.google/cirq/tutorials/aqt/start)|Trapped ions|
+|[Pasqal](https://quantumai.google/cirq/tutorials/pasqal/start)|Neutral atoms|
+|[IonQ](https://quantumai.google/cirq/tutorials/ionq/getting_started)|Trapped ions|
+|[Microsoft Azure Quantum](https://quantumai.google/cirq/tutorials/azure-quantum/getting_started_ionq)|Trapped ions (Honeywell and IonQ)|
 
 
 ## High performance quantum circuit simulators
 
 |Name|Main sponsor|Description|
 |--- |--- |--- |
 |[qsim](https://github.com/quantumlib/qsim)|Google|An open-source, high performance circuit simulator for Schrödinger simulations|
-|[qflex](https://github.com/ngnrsaa/qflex)|NASA|An open-source, high performance circuit simulator for Feynman-based simulations optimized for GPUs|
 |[quimb](https://github.com/jcmgray/quimb)|Johnny Gray|An open-source, high performance circuit simulator using tensor-networks|
```

Comment:
Out of curiosity: Why remove qflex?

Reply:
cc: @95-martin-orion

Reply:
The motivation for this was to focus communications around qsim as the "standard" simulator for Cirq. Separately, while qFlex was originally Cirq-compatible, it hasn't been maintained - I'm doubtful that it works with modern Cirq.

Reply:
+1 thanks Orion